patches/react-vtree+3.0.0-beta.1.patch

```diff
@@ -3,127 +3,100 @@ index 5e7f57e..5e764e9 100644
 --- a/node_modules/react-vtree/dist/es/Tree.d.ts
```

Comment:
Change from running `git add --renormalize . ` after adding .gitattributes file to normalize line endings